thyrel-web/src/components/Home/AvatarCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { Avatar, Icon } from 'rsuite';
+import { baseColor } from '../../styles/colors';
+import Box from '../../styles/Box';
+// import styled from '@emotion/styled';
+import { css } from '@emotion/css';
+
+type AvatarCardProps = {
+  image?: string;
```

Comment:
```suggestion
  image: string;
```

Si y'a pas d'image le component n'est pas très utile, on peut la marquer en `required` donc